### PR TITLE
resolving configuration issue MB-1715

### PR DESCRIPTION
**Pull request by ayyoob**
https://wso2.org/jira/browse/MB-1715

### CODE REVIEW

File: components/andes/org.wso2.carbon.andes.authentication/src/main/java/org/wso2/carbon/andes/authentication/andes/oauth/OAuthTokenValidaterStubFactory.java [hunk 1]
```diff
@@ -99,8 +99,8 @@ private OAuth2TokenValidationServiceStub generateStub() throws OAuthTokenValidat
 					auth.setPreemptiveAuthentication(true);
 					String username = config.getUsername();
 					String password = config.getPassword();
-					auth.setPassword(username);
-					auth.setUsername(password);
```
Comment (a5anka) [inline components/andes/org.wso2.carbon.andes.authentication/src/main/java/org/wso2/carbon/andes/authentication/andes/oauth/OAuthTokenValidaterStubFactory.java]: Does it make any difference, changing the order of calls? Just trying to understand :smile:

Comment (ayyoob) [inline components/andes/org.wso2.carbon.andes.authentication/src/main/java/org/wso2/carbon/andes/authentication/andes/oauth/OAuthTokenValidaterStubFactory.java]: In here username is set to password and password is set to username, its not about the order

Comment (a5anka) [inline components/andes/org.wso2.carbon.andes.authentication/src/main/java/org/wso2/carbon/andes/authentication/andes/oauth/OAuthTokenValidaterStubFactory.java]: :+1: got it

**Comment by ramith** [comment]
Could you create a jira and put that in the PR description? @ayyoob

**Pull request merged**